helpers/test-helper.js

```diff
@@ -90,14 +90,14 @@ async function checkError(promise, errMsg, isAssert) {
   assert.equal(receipt.status, 0x00);
 
   if (isAssert) {
-    const network = await web3GetNetwork();
     const transaction = await web3GetTransaction(tx);
-    if (network !== "coverage") {
-      // When a transaction `throws`, all the gas sent is spent. So let's check that we spent all the gas that we sent.
-      // TODO: Properly account for EIP150 and passing only 63/64 gas on CALL.
-      // See https://github.com/JoinColony/colonyNetwork/issues/194 for details.
-      assert.closeTo(transaction.gas, receipt.gasUsed, 95000, "didnt fail - didn't throw and use all gas");
-    }
+    // As of EIP150, when a CALL is made only 63/64 on the remaining gas is passed on to to the called contract.
+    // If an assert fails, that gas is expected to be spent.
+
+    const remainingBeforeCall = (transaction.gas - receipt.gasUsed) * 64;
```

Comment:
Sorry to keep picking at this but here I have no idea what's going on each line of gas calculation. Please make it obvious with the variables naming. For example, this is called `remainingBeforeCall` while the calculation is actually the total gas sent, minus the used so far multiplied by 64?!

Reply:
`receipt.gasUsed` will be gas used in `EtherRouter` + 63/64 of the remaining, so `transaction.gas - receipt.gasUsed` will be the remaining 1/64, then i multiply it by 64. So the result will be the remaining after `EtherRouter` has done its job.

This is not final solution, just me debugging and trying to extract the values i need. I accidentally pushed this. If i am wasting your time here, then i can call you when i finish.

Reply:
I had a chat with @area about this and we might just remove the assert checks after all. (sorry) We will likely use asserts in code but should not be able to get to them while testing.